.gitlab-ci.yml

```diff
@@ -14,6 +14,8 @@ include:
     file: '.gitlab-ci-check-docker-build.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-docker-deploy.yml'
 
 # Function ported from template gitlab-ci-check-docker-build
 .export_docker_vars: &export_docker_vars |
```

Comment:
We don't need this any more.

Reply:
sorry @lluiscampos, which thing we don't need anymore?

Reply:
understood, the `export_docker_vars`